internal/backend/triggers/triggers.go

```diff
@@ -27,21 +31,75 @@ func (m *triggers) Create(ctx context.Context, trigger sdktypes.Trigger) (sdktyp
 	}
 
 	trigger = trigger.WithNewID()
+	if trigger.ConnectionID() == fixtures.BuiltinSchedulerConnectionID {
```

Comment:
shouldn't this be after the actual trigger creation?

Reply:
i guess you worry something will happen between here and line 44. please comment about it.

Reply:
no, first I want to create a scheduler workflow 
1. I need to save scheduleID in trigger
2. I want to report an error if creation of schedule workflow failed and not to create a trigger